### ISSUE TITLE
is that the official bibutils repository?

### ISSUE THREAD

**Issue by monperrus** [wilx/bibutils#1]
Hi all, I'm looking for the official bibutils repository, or the most active fork of bibutils if the main repo is dead.

Is that the official bibutils repository?

**Comment by wilx** [comment]
No. Bibutils source does not have a public version controlled repository. The author has a [project on SourceForge](https://sourceforge.net/projects/bibutils/) where does releases, so that is an official Bibutils home, AFAIK.

This repository is an attempt to reconstruct Bibutils' history. It was forked and is based on work done in [jayvdb/bibutils-archive](https://github.com/jayvdb/bibutils-archive).

**Comment by monperrus** [comment]
Thanks.

What do you recommend to contribute back with some changes? A pull-request here? On another repo?

**Comment by wilx** [comment]
[**@monperrus**](https://github.com/monperrus) commented on [May 18, 2018, 11:07 AM GMT+2](https://github.com/wilx/bibutils/issues/1#issuecomment-390145339 "2018-05-18T09:07:51Z - Replied by Github Reply Comments"):
> Thanks.
> 
> What do you recommend to contribute back with some changes? A pull-request here? On another repo?

I have contributed bits by creating a discussion topic in https://sourceforge.net/p/bibutils/discussion/general/ and attaching diffs to it.

**Comment by wilx** [comment]
I see you have already tried there. I am closing this. Good luck.

**Comment by monperrus** [comment]
thanks a lot for your help.

**Comment by monperrus** [comment]
In order to create a diff, does somebody already maintain a Github branch with the latest commits from Sourceforge?

**Comment by wilx** [comment]
This repository has version 6.2 code and this version is the latest available from official SourceForge project site. I think you can use this.
